README.md

```diff
@@ -1,2 +1,139 @@
-# CODE9000
-Urban Data Birds: Providing a new haven for urban biodiversity.
+\#Code9000
+===================
+----------
+
+\#Code9000 is a project in which Digipolis Ghent and the City of Ghent experiment with open data to create new open source proof-of-concepts. We’ve been doing this since 2011 and love to work with technical, business and communication students to co-create new tools. Students get real cases to work on and move beyond the theoretical and we get receive new learnings and code to test and iterate upon. This time we want to build an IOT use case, combining hardware and software to feed our data portal rather than using the data we already have.  
+
+
+Getting Started
+-------------
+----------
+####  Requirements
```

Comment:
The requirements are shuffled per project. Like PHP, Composer and Mysql is for the API

Also, do we only need python for the Hardware side? I guess we'll need more like the Raspian. CC @DylanVanAssche